pipelines/tasks/CollectMultiplePicardMetrics.wdl

```diff
@@ -0,0 +1,28 @@
+
+task CollectMultipleMetrics {
+  File aligned_bam
+  File ref_genome_fasta
+  String output_filename
+
+  command {
+    java -Xmx3g -jar /usr/picard/picard.jar CollectMultipleMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      METRIC_ACCUMULATION_LEVEL=ALL_READS \
+      INPUT="${aligned_bam}" \
+      OUTPUT="${output_filename}" \
+      PROGRAM=CollectAlignmentSummaryMetrics \
+      PROGRAM=CollectInsertSizeMetrics \
+      PROGRAM=CollectGcBiasMetrics \
+      REFERENCE_SEQUENCE="${ref_genome_fasta}"
+
+      tar -czvf "${output_filename}.tar.gz" "${output_filename}"*
+  }
+  runtime {
+    docker:"humancellatlas/picard:2.10.10"
```

Comment:
Don't know if you want the R-dependencies, if you need, `humancellatlas/picard:2.10.10-7ab16db` is the one with R-base and ggplot2 installed.

Reply:
I am going to pull Jishu's task in after she merges her picard task; at that point I will change the docker.